caffe2/python/onnx/backend.py

```diff
@@ -408,6 +434,16 @@ def _create_rnn_variant(cls, init_model, pred_model, n, opset_version):
                                  pred_model.graph.value_info):
             if x.name == W:
                 input_size = x.type.tensor_type.shape.dim[2].dim_value
+                e_type = x.type.tensor_type.elem_type
+                if e_type == TensorProto.FLOAT16:
```

Comment:
don't we have some helper to do this conversion? or can you make one?

Reply:
Yeah, I can do that.